### DISCUSSION TITLE
Show all databases options is not showing anymore

### DISCUSSION THREAD

**Discussion by paramThakkar1505** [dbeaver/dbeaver#23099] [Q&A]
### Description

I am connecting to RDS cluster which is of PostgreSQL and after the update, Show All Databases options is not there under the PostgreSQL tab

### DBeaver Version

24.0.0

**Comment by LonwoLonwo** [comment]
Hello @paramThakkar1505 

 You can find this setting on the main connection page now.

![2024-03-08 11_30_34-Connect to a database](https://github.com/dbeaver/dbeaver/assets/45152336/21e7996c-9627-41b8-9301-a9dee80ad64e)

**Comment by paramThakkar1505** [comment]
Okay got it. Thanks.
